openedx/core/lib/block_structure/manager.py

```diff
@@ -110,4 +111,5 @@ def clear(self):
         Removes cached data for the block structure associated with the given
         root block key.
         """
+        print "clear() in manager.py ***************"
```

Comment:
By the way, there are log statements for this within the block_structure_cache code.